### ISSUE TITLE
Should I tune the default parameters if I reduce the batchsize to 4 for deblurring task?

### ISSUE THREAD

**Issue by ChefLiutao** [swz30/MPRNet#50]
Using 4 GPUs of 2080Ti to train the MPRNet deblurring model, I find the maximum batchsize is 4 while the default batchsize is 16. So, other default parameters such as learning rate need to be tuned accordingly ?

**Comment by adityac8** [comment]
Hi @ChefLiutao 

Please refer to https://github.com/swz30/MPRNet/issues/15#issuecomment-792654636
We did not try with batch size 4.

Thanks
